### PR TITLE
Add adding and removing features

### PR DESCRIPTION
**Pull request by danalvrz**
- Removed all hardcoded items from array.
- Created addRemove.js for new functionalities.
- Implemented functions for adding, editing, and deleting tasks.
- Implemented function for clearing all completed tasks from list.

### CODE REVIEW

File: src/addRemove.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      editBoxes[i].parentNode.classList.remove('editable');
+    });
+  }
+};
+
+const deleteTask = () => {
+  const trashCans = document.querySelectorAll('.trashCan');
+  for (let i = 0; i < trashCans.length; i += 1) {
+    trashCans[i].addEventListener('click', () => {
+      // eslint-disable-next-line max-len
```
Comment (Dipeshtwis) [inline src/addRemove.js]: - [x] I would not recommend disabling linters instead, try to improve the code. Here you can complete the line in 2 or more. It doesn't need to be in one line if the line is long.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @danalvrz, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: The application is working fine.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

- [x] README file should have a proper description of the project even if it's one or two-line so that anyone can understand your project. Also, write all the steps from cloning to run the project locally.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/addRemove.js (hunk 1 above)
Comment (danalvrz) [reply src/addRemove.js]: Thanks for the comments! I fixed it

**Comment by omar-labana** [comment]
Hi @danalvrz, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**